src/components/CollapsibleTable.js

```diff
@@ -114,6 +117,22 @@ const StyledTableCell = styled(TableCell)(() => ({
   },
 }));
 
+const TruncatedTableCell = styled(StyledTableCell)(() => ({
```

Comment:
Instead of expanding on hover, can we make the ellipsis a button that can be clicked to reveal more of the text?

Reply:
Thanks for the suggestion! I discussed this with @solideoglori and he said he might prefer the hover option for now.